xbmc/windowing/android/AndroidUtils.cpp

```diff
@@ -251,7 +251,7 @@ bool CAndroidUtils::SetNativeResolution(const RESOLUTION_INFO &res)
     CXBMCApp::SetDisplayMode(atoi(res.strId.c_str()));
     s_res_cur_displayMode = res;
   }
-  else if (abs(currentRefreshRate() - res.fRefreshRate) > 0.0001)
+  else if (std::abs(currentRefreshRate() - res.fRefreshRate) > 0.0001)
```

Comment:
OK, `std::abs` is fine. You should include cmath in case another include is removed that breaks the code.